src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -55,29 +53,83 @@ public void sendObjectionSubmittedCustomerEmail (
             String companyNumber,
             Objection objection
     ) throws ServiceException {
+
         CompanyProfileApi companyProfile = companyProfileService.getCompanyProfile(companyNumber, requestId);
 
         String companyName = companyProfile.getCompanyName();
+        Map<String, Object> data = constructEmailDataMap(companyName, companyNumber, objection);
+
         String emailAddress = ericHeaderParser.getEmailAddress(ericAuthorisedUser);
-        Map<String, Object> data = new HashMap<>();
+        EmailContent emailContent = constructEmailContent(EmailType.CUSTOMER,
+                requestId, emailAddress, data);
 
-        data.put("company_name", companyName);
-        data.put("company_number", companyNumber);
-        data.put("objection_id", objection.getId());
-        data.put("reason", objection.getReason());
-        data.put("attachments", objection.getAttachments());
+        logger.debugContext(requestId, "Calling Kafka client to send customer email");
+        kafkaEmailClient.sendEmailToKafka(emailContent);
+        logger.debugContext(requestId, "Successfully called Kafka client");
+    }
+
+    @Override
+    public void sendObjectionSubmittedDissolutionTeamEmail(
+            String requestId,
+            String companyNumber,
+            Objection objection
+    ) throws ServiceException {
 
-        EmailContent emailContent = new EmailContent.Builder()
-                .withOriginatingAppId(originatingAppId)
+        CompanyProfileApi companyProfile = companyProfileService.getCompanyProfile(companyNumber, requestId);
```

Comment:
I think the companyProfile should be called in the ObjectionService and passed into the EmailService, as it is we do the same profile call twice, once for external once for internal.

Reply:
Now passing company profile

Reply:
I think Ian meant to pass a CompanyProfile rather than the Api. But if we only need the company name (company number is already in the Objection) then we can just have the company name as a String input param to this method.

Reply:
Sorry, CompanyProfileApi is a bad name! (i know it's not ours) ok I can see it's a pojo, not the api. But I think we only need the company name instead of the whole profile.

Reply:
Changed CompanyProfile for companyName for customer emails.